host_core/lib/host_core/cloud_event.ex

```diff
@@ -1,7 +1,6 @@
 defmodule HostCore.CloudEvent do
-  def new(data, event_type) do
+  def new(data, event_type, host \\ HostCore.Host.host_key()) do
```

Comment:
Is this `\\` syntax declaring `host` optional, and if it's not provided then it's assigned the value on the right hand side? (if so, elixir you're cool)

Reply:
That's exactly what this is.. and because it's the last parameter in line and has a default, you can leave it off in all other calls where it isn't needed.